source/gateway_test.go

```diff
@@ -246,47 +245,3 @@ func TestIsDNS1123Domain(t *testing.T) {
 		})
 	}
 }
-
-func TestDualStackLabel(t *testing.T) {
```

Comment:
The test is removed, which is understandable, but what about a test where ipv4 with ipv6 ?

Reply:
This source does not seem to go into details about ipv4 and ipv6 in testing. The test I removed was purely about if or not the annotation was properly attached and producing a label or not.